### PR TITLE
HCIDOCS-140: Reboot after coreos-installer when adding worker nodes

### PR DESCRIPTION
**Pull request by tshwartz**
Version(s):

Issue:

Link to docs preview:

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/sno-adding-worker-nodes-to-sno-clusters-manually.adoc [hunk 1]
```diff
@@ -158,13 +158,17 @@ where:
...
+.. For networks that have DHCP enabled, you do not need to set a static IP. Instead, do the following: 
+
+... Run the following `coreos-installer` command from the target host console to install the system:
 +
 [source,terminal]
 ----
 $ coreos-installer install --ignition-url=<hosted_worker_ign_file> <hard_disk>
 ----
 
+... If the installation hangs instead of proceeding automatically, manually reboot the worker node.
```
Comment (tsorya) [inline modules/sno-adding-worker-nodes-to-sno-clusters-manually.adoc:170]: not sure i understand , coreos-installer command doesn't reboot the node as far as i remember so user will need to reboot it manually in anycase. Though command itself must finish.

Comment (tshwartz) [reply modules/sno-adding-worker-nodes-to-sno-clusters-manually.adoc:170]: Thanks @tsorya . My understanding is that: 
- The command doesn't always finish, but sometimes nothing happens after it is entered. In which case the reboot is needed. 
- This happens in some platforms and not in others. (Less relevant.)

@anildhingra is my understanding correct? 

If so, I can change 'If the installation hangs' to 'If the command hangs" in the second bullet. If not, I can change it to "Wait for the command to complete, and then reboot the worker node". 
Thanks in advance.

Comment (tsorya) [reply modules/sno-adding-worker-nodes-to-sno-clusters-manually.adoc:170]: Didn't know we have such an issue. We run same logic in assisted-installer installation and never saw such issue. I just wonder how we can tell user that command actually wrote rhcos to the disk? I assume "hangs" means logging was stopped on 100% ?

**Comment by tshwartz** [comment]
Following a discussion with @anildhingra and @tsorya, it was decided to discard and close this story.

**Pull request closed without merging**